### ISSUE TITLE
Blog post: Pangeo hub deployment

### ISSUE THREAD

**Issue by choldgraf** [2i2c-org/team-compass#244]
### Description

We've recently wrapped up the initial deployment of the 2i2c Pangeo Hub 🎉 

Getting this deployment done was quite an effort that exposed a lot of complexity that wasn't obvious ahead of time. It was a good example of how much care needs to go into crafting cloud infrastructure and operating it over time.

We should write a blog post that briefly describes the major issues that we ran into here, how we got over them, and what kinds of lessons others could learn from the experience.

### Value / benefit

- Other hub engineers would benefit by learning from our experience, so that they might be able to more easily anticipate and get over challenges associated with deploying a Pangeo hub.
- People interested in 2i2c would potentially see this blog post as a demonstration of our expertise, as well as the complexity that 2i2c can handle for others
	

### Implementation details

It'd be helpful if we kept track of where major discussion happened around this deployment, so that we can use this to generate ideas for topics to cover. A few that I know of:

- Auto-provisioning discussion: https://github.com/2i2c-org/pilot-hubs/issues/666#issuecomment-915956414
- Challenges around NFS and Google Filestore expense: https://github.com/2i2c-org/pilot-hubs/pull/651
- Requester pays data: https://github.com/2i2c-org/pilot-hubs/issues/655
- Private nodes: https://github.com/2i2c-org/pilot-hubs/pull/538
- GitHub Teams auth: https://github.com/2i2c-org/pilot-hubs/issues/598

### Tasks to complete

- [ ] Continue keeping track of updates as we make them
- [ ] Once we are happy with the initial deployment, write up an update post!

**Comment by sgibson91** [comment]
Might I suggest this be done in at least 2 parts? We've learned a lot and trying to compress it all into a single blog is probably a reach!

**Comment by choldgraf** [comment]
Makes sense to me - i just wanted this issue to track the stuff we might talk about 🙂
